client/scripts/views/pages/view_proposal/create_comment.ts

```diff
@@ -215,37 +215,44 @@ const CreateComment: m.Component<{
             }),
```

Comment:
This file just involves a slight UI layout change so that new comment errors are visually aligned + spaced.

Reply:
👍